src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/EmbeddedLanguages/VirtualChars/VirtualCharSequence.cs

```diff
@@ -180,13 +184,65 @@ public VirtualChar this[int index]
     public VirtualChar? Find(int position)
         => _sequence.Find(_tokenStart, position);
 
-    public bool Contains(VirtualChar @char)
-        => IndexOf(@char) >= 0;
+    /// <inheritdoc cref="VirtualCharGreenSequence.IsDefault"/>
+    public bool IsDefault => _sequence.IsDefault;
```

Comment:
note: we'll still need this (sicne it is effectively the null-check on a struct).  But i will likely be removing isEmpty and IsDefaultOrEmpty as string itself doesn't have those.